setup.cfg

```diff
@@ -54,7 +54,7 @@ test =
 ; release scripts and process.
 dev_release =
     towncrier ~= 19.2
-    pydoctor ~= 21.2; python_version >= "3.6"
+    pydoctor ~= 21.2.2; python_version >= "3.6"
```

Comment:
It will install 21.2.2 if nothing was installed yet, but I don't know if it forces an update if an older version was already installed.

In any case, we need 21.2.2 or higher for the docs to be rendered correctly, so I think `~= 21.2.2` is a more accurate description of the requirements even if it doesn't make much of a difference in practice.

The `python_version >= "3.6"` part is only needed for the release branch, since `trunk` is now 3.6+. But it's harmless to have it on `trunk`.